packages/graphback-core/src/runtime/withSubscriptionFilter.ts

```diff
@@ -0,0 +1,19 @@
+import { withFilter, ResolverFn } from 'graphql-subscriptions'
+
+export function withSubscriptionFilter(asyncIterator: ResolverFn, filter: any, subscriptionName: string): ResolverFn {
+  if (!filter) {
+    return asyncIterator
+  }
+
+  return withFilter(asyncIterator, (payload: any) => {
+    const subscriptionPayload = payload[subscriptionName]
+
+    for (const key of Object.keys(filter)) {
+      if (!subscriptionPayload[key] || subscriptionPayload[key].toString() !== filter[key].toString()) {
+        return false
+      }
```

Comment:
Nice. Seems like we only support equality filtering. Do we have plans to extend this?

Reply:
Also if you guys can come back to us with desirable GraphQL filter for subscriptions so we can adjust spec.

Reply:
Agreement that we can extend this, but in a follow up issue/PR.

Reply:
Sure, let's discuss these in separate issue.

Reply:
https://github.com/aerogear/graphback/issues/1760

Reply:
Thanks